app/components/forms/wizard/basic-details-step.js

```diff
@@ -52,6 +52,9 @@ export default Component.extend(FormMixin, EventWizardMixin, {
     return this.get('data.event.socialLinks').filterBy('isDeleted', false);
   }),
 
+  isUserUnverified: computed(function() {
+    return (this.authManager.currentUser.isVerified ? false : true);
```

Comment:
```js
isUserUnverified: computed('authManager.currentUser.isVerified', function() {
return !this.authManager.currentUser.isVerified
})
```

@uds5501

Reply:
@CosmicCoder96 Completed the changes as requested!